gpkitmodels/GP/aircraft/prop/propeller.py

```diff
@@ -0,0 +1,60 @@
+" propeller model "
+from numpy import pi
+from gpkit import Model, parse_variables, SignomialsEnabled, SignomialEquality
+
+class Propeller(Model):
+    """ Propeller Model
+
+    Variables
+    ---------
+    R          10           [m]         prop radius
+    W          10           [lbf]        prop weight
+
+    """
+    def setup(self):
+        exec parse_variables(Propeller.__doc__)
+
+    def performance(self,state):
```

Comment:
Instead of declaring it as a model I would declare like: 
```python 
"""
docstring
"""

performance = Propeller_Performance

def setup(self):
...
```
Also can we rename it from `performance` to `flight_model` so it's consistent with `wing.py` and the other GP models in this repo?